src/content/changes/highlights/2024-07-21.md

```diff
@@ -0,0 +1,186 @@
+operator/compare/v0.6.0...v0.7.0)
+        - Add support for Alloy as logging agent Add `--logging-agent` flag to toggle between Promtail and Alloy
+- [loki-app](https://github.com/giantswarm/loki-app)
+    - [0.21.0](https://github.com/giantswarm/loki-app/compare/v0.20.2...v0.21.0)
+        - Upgraded upstream chart from 6.6.4 to 6.7.1 - see [changelog](https://github.com/grafana/loki/blob/main/production/helm/loki/CHANGELOG.md) for more information.
+        - Upgrade loki from 3.0.0 to 3.1.0
+- [object-storage-operator](https://github.com/giantswarm/object-storage-operator)
+    - [0.8.0](https://github.com/giantswarm/object-storage-
+operator/compare/v0.7.0...v0.8.0)
+        - ReclaimPolicy added in the Bucket CR to manage the data clean up (retain or delete).
+        - Add a finalizer on the Azure secret to prevent its deletion.
+        - Empty all the objects in the S3 bucket in case of bucket deletion.
+- [observability-bundle](https://github.com/giantswarm/observability-bundle)
+    - [1.5.1](https://github.com/giantswarm/observability-
+bundle/compare/v1.5.0...v1.5.1)
+        - Upgrade `prometheus-operator-crd` to 11.0.1.
+    - [1.5.0](https://github.com/giantswarm/observability-
+bundle/compare/v1.4.0...v1.5.0)
+        - Add `alloy` v0.3.0 as `alloy-logs`
+        - prometheus-operator will not check promql syntax for prometheusRules that are labelled `application.giantswarm.io/prometheus-rule-kind: loki`
+- [observability-operator](https://github.com/giantswarm/observability-operator)
+    - [0.3.0](https://github.com/giantswarm/observability-
+operator/compare/v0.2.0...v0.3.0)
+        - Delete monitoring resources if monitoring is disabled at the installation or cluster level using the giantswarm.io/monitoring label.
+- [prometheus-operator-crd](https://github.com/giantswarm/prometheus-operator-crd)
+    - [11.0.1](https://github.com/giantswarm/prometheus-operator-
+crd/compare/v11.0.0...v11.0.1)
+        - Add `helm.sh/resource-policy: keep` annotation to all CRDs to avoid deletion during Helm operations.
+- [prometheus-rules](https://github.com/giantswarm/prometheus-rules)
+    - [4.8.0](https://github.com/giantswarm/prometheus-
+rules/compare/v4.7.0...v4.8.0)
+        - Move alloy to monitoring namespace
+    - [4.7.0](https://github.com/giantswarm/prometheus-
+rules/compare/v4.6.3...v4.7.0)
+        - Support for loki rules to management clusters in alloy config
+        - grafana datasource for MC loki ruler
+        - Make dns-operator-azure capz only.
+        - Fix PromtailDown alert to fire only when the node is ready.
+- [kube-downscaler-app](https://github.com/giantswarm/kube-downscaler-app)
+    - [0.3.0](https://github.com/giantswarm/kube-downscaler-
+app/compare/v0.2.0...v0.3.0)
+        - Push `kube-downscaler` app to all collections.
+    - [0.2.0](https://github.com/giantswarm/kube-downscaler-
+app/compare/v0.1.0...v0.2.0)
+        - Added `enabled` field in values to disable whole chart if needed.
```

Comment:
@Rotfuks These are from Atlas. Please adapt the content to the standard format (name, version, description)
[example](https://raw.githubusercontent.com/giantswarm/docs/main/src/content/changes/highlights/2024-02-15.md)